src/idpi/products/pot_vortic.py

```diff
@@ -0,0 +1,68 @@
+# Standard library
+import sys
+import logging
+
+# Third-party
+import numpy as np
+
+# First-party
+from idpi.operators.destagger import destagger
+from idpi.operators.pot_vortic import fpotvortic
+from idpi.operators.rho import f_rho_tot
+from idpi.operators.theta import ftheta
+from idpi.operators.total_diff import TotalDiff
+from idpi.operators.vertical_interpolation import interpolate_k2p
+from idpi.operators.vertical_interpolation import interpolate_k2theta
+from idpi.operators.vertical_reduction import integrate_k
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def _compute_pot_vortic(U, V, W, T, P, QV, QC, QI, HHL, theta):
+    logger.info("Computing total density")
```

Comment:
I'm lacking a global view of the project but since this is the only product, this felt like an appropriate granularity. The log events can be demoted to debug when appropriate.

Reply:
As pointed out here: https://github.com/MeteoSwiss-APN/icon_data_processing_incubator/pull/54/files#r1214583863
the main pattern will be a stream of data in the time dimension lead time by lead time. 
Whatever we do, I think we should have the same pattern and logging for all operators. I think it would make sense to create a design for it so that all operator log in the same manner, like using a decorator that will register any operator in the logging. I would do that in a separate design and PR